pkg/kubectl/resource_printer.go

```diff
@@ -600,15 +641,6 @@ func printPodBase(pod *api.Pod, w io.Writer, withNamespace bool, wide bool, show
 	return err
 }
 
-func printPodList(podList *api.PodList, w io.Writer, withNamespace bool, wide bool, showAll bool, columnLabels []string) error {
```

Comment:
They were moved without changes, the `PrintObj` method of `HumanReadablePrinter` is a little misplaced. Reverted.